src/pages/api/fetch-people.ts

```diff
@@ -0,0 +1,130 @@
+import type { NextApiRequest, NextApiResponse } from 'next';
```

Comment:
best pattern in next.js is to avoid coding business logic inside the api route directly, otherwise you will block yourself from doing SSR.

better to do business logic in a separate function and import it, such as (simplified):

```
export default async function (req, res) {
  const data = zod.validate(req.query)

  try {
    const result = await myLibFn(data)
    res.status(200).json(result)
  } catch (err) {
    res.status(500).json({ err })
  }
}
```